src/include/distributed/maintenanced.h

```diff
@@ -0,0 +1,18 @@
+/*-------------------------------------------------------------------------
+ *
+ * maintenanced.h
+ *
+ * Copyright (c) 2017, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
```

Comment:
sorry, put 2 comments in 1.

1: maintenance.d should have a meaningful comment at the top
2: shared_library_init.h should have a copyright notice.